drivers/bridge/port_mapping.go

```diff
@@ -5,13 +5,15 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"strings"
 
 	"github.com/docker/libnetwork/types"
 	"github.com/sirupsen/logrus"
 )
 
 var (
-	defaultBindingIP = net.IPv4(0, 0, 0, 0)
+	defaultBindingIP   = net.IPv4(0, 0, 0, 0)
+	defaultBindingIPV6 = net.ParseIP("::1")
```

Comment:
Are you sure about `::1`? This is localhost in IPv6 where 0.0.0.0 means any address in IPv4.

Reply:
Ah, you are right. This should be `::`.

Reply:
Fixed.

Reply:
:man_dancing: awesome! :+1: